components/authz-service/server/v2/rules_property_test.go

```diff
@@ -15,14 +15,17 @@ import (
 	"google.golang.org/grpc/codes"
 
 	api "github.com/chef/automate/api/interservice/authz/v2"
+	constants_v2 "github.com/chef/automate/components/authz-service/constants/v2"
 	"github.com/chef/automate/components/authz-service/testhelpers"
+	"github.com/chef/automate/lib/grpc/auth_context"
 	"github.com/chef/automate/lib/grpc/grpctest"
 )
 
 const (
-	idRegex        = "^[a-z0-9-_]{1,64}$"
-	conditionLimit = 10 // help avoid the grpc limit of 4194304
-	projectLimit   = 6  // system limitation
+	idRegex          = "^[a-z0-9-_]{1,64}$"
+	conditionLimit   = 10 // help avoid the grpc limit of 4194304
+	projectLimit     = 6  // system limitation
+	SuperuserSubject = "tls:service:deployment-service:internal"
```

Comment:
Copied from postgres test... since it essentially uses the same setup it allows for the same superuser.